project-base/storefront/components/Layout/CustomerLayout.tsx

```diff
@@ -0,0 +1,47 @@
+import { UserNavigation } from 'components/Blocks/UserNavigation/UserNavigation';
+import { Breadcrumbs } from 'components/Layout/Breadcrumbs/Breadcrumbs';
+import { CommonLayout, CommonLayoutProps } from 'components/Layout/CommonLayout';
+import { Webline } from 'components/Layout/Webline/Webline';
+import { useDomainConfig } from 'components/providers/DomainConfigProvider';
+import { TypeBreadcrumbFragment } from 'graphql/requests/breadcrumbs/fragments/BreadcrumbFragment.generated';
+import useTranslation from 'next-translate/useTranslation';
+import { useIsUserLoggedIn } from 'utils/auth/useIsUserLoggedIn';
+import { getInternationalizedStaticUrls } from 'utils/staticUrls/getInternationalizedStaticUrls';
+import { twMergeCustom } from 'utils/twMerge';
+
+type CustomerLayoutProps = {
+    pageTitle?: string;
+    breadcrumbs?: TypeBreadcrumbFragment[];
```

Comment:
I think `breadcrumbs` are part of `CommonLayoutProps`, right?

Reply:
Yes, good point, I forgot to delete because it's copy paste from breno